lua/lean/stderr.lua

```diff
@@ -0,0 +1,19 @@
+local log = require'vim.lsp.log'
+local stderr = {}
+
+-- Show stderr output in :messages
+-- TODO: add upstream neovim API
+function stderr.enable()
+  local old_error = log.error
+  log.error = function(...)
+    local argc = select('#', ...)
+    if argc == 0 then return true end -- always enable error messages
+    if argc == 4 and select(1, ...) == 'rpc' and select(3, ...) == 'stderr'
+        and string.match(select(2, ...), 'lean') then
+      print(select(4, ...))
```

Comment:
Perhaps `vim.api.nvim_err_writeln` instead of `print`?

Reply:
Ok, changed!